src/scss/components/switch/mixins/_hxSwitchControl.scss

```diff
@@ -0,0 +1,154 @@
+//
+// ===== Mixins for <hx-switch-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __switchControl($pseudo-state: null, $checkBox-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+  } @else if $pseudo-state == hover {
+    @if $checkBox-state == unchecked {
```

Comment:
Please indent this block.

Reply:
As per our lint rules, `yarn lint` is not throwing any error. So, i continued the same.